Exercises-3-Basic/src/exercise/Number3.java

```diff
@@ -0,0 +1,34 @@
+package exercise;
+
+//Write a Java program to test if an array contains a specific value.
+//Example,
+//Input: An array of integers [2 , 5 , 1, 4, 10]   ,  4
+//Output: Should print "Yes", as the array contains the number 4.
+//
+//Input: An array of integers [2 , 5 , 1, 4, 10]   ,  3
+//Output: Should print "No", as the array does not contain the number 3.
+public class Number3 {
```

Comment:
The solution is correct!
Let's try to evolve this solution a little bit.
Ask the user to provide you the array's elements and the number to find.

Reply:
Okay